### DISCUSSION TITLE
hack or hi3518

### DISCUSSION THREAD

**Discussion by Antarctic74** [roleoroleo/yi-hack-MStar#323] [General]
Please take in consideration to release the hack for hi3518 chipset too!  the crypto ones lacks rtsp support.... please!

**Comment by roleoroleo** [comment]
https://github.com/alienatedsec/yi-hack-v4

**Reply by Antarctic74** [reply]
RTSP still to buy?

**Reply by roleoroleo** [reply]
No.
https://github.com/alienatedsec/yi-hack-v5
